src/main/java/org/recommendation/service/SimilaratiyRating.java

```diff
@@ -0,0 +1,38 @@
+package org.recommendation.service;
+
+
+import org.recommendation.model.Rating;
+
+import java.util.List;
+
+abstract public class SimilaratiyRating {
```

Comment:
why this class is abstract ?

Reply:
getSimilarRatings can be implemented for both user and movie for now we have use case of user only but in case we want to extend then we can leverage this.